cypress-io/cypress
# feat: show tooltips indicating the source of config values

- Closes https://github.com/cypress-io/cypress/issues/21830

### User facing changelog


Fix an issue were user does not see a tooltip showing the source of the config value on hover. Fixes #21830

### Additional details
In 9.x, in addition to the legend, we show tooltips indicating the source of values in the in the resolved config, when you hover over the values. We should match this behavior in 10.0.

### Steps to test
- Prepare/view target project
- Go to project settings page
- In the `Resolved Configuration` section, see a list of configs for the project, listed in a { key: value } pair approah.
- Hover on the value
- Verify that a tooltip is shown which indicates the source of the config value.

### How has the user experience changed?
BEFORE

https://user-images.githubusercontent.com/45232708/184300205-5052b8fe-f7d8-4499-a45c-c5e074b2d976.mov



AFTER

https://user-images.githubusercontent.com/45232708/184299635-325b64c5-7806-4c65-93f4-93ea75a4e6fc.mov



### PR Tasks
